### PR TITLE
Fixed typo in _OneToOneFeatureMixin

### PR DESCRIPTION
**Pull request by amanabdullayev**
Importing several sklearn functions were failing due to typo in `_OneToOneFeatureMixin`, which is currently `OneToOneFeatureMixin` (without the underscore) in sklearn.

### CODE REVIEW

**Comment by larun1916** [comment]
Is it fixed? Can it be used?

**Comment by amanabdullayev** [comment]
@larun1916, I have fixed it but waiting for @Garve to merge the fix. Once it is merged then you can use it. but if you want to use it ASAP you can follow the steps:
- fork the repo to your github, 
- then clone the repo to your local machine
- just change _OneToOneFeatureMixin => to OneToOneFeatureMixin in several py files.
- in your jupyter notebook: 
```
import sys

sys.path.append(
    "{{path to your local repo}}"
)
```

then you can import mamimo as usual and use it, e.g.

```
import pandas as pd
from mamimo.carryover import ExponentialCarryover
from mamimo.saturation import ExponentialSaturation
from mamimo.linear_model import LinearRegression
from mamimo.analysis import breakdown
from mamimo.time_utils import PowerTrend
from sklearn.compose import ColumnTransformer
from sklearn.pipeline import Pipeline
from sklearn.preprocessing import OneHotEncoder
import matplotlib.pyplot as plt
```

**Review by RobKuebler — APPROVED** [review]
Thanks a lot!

**Pull request merged**
